Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent ant
+    stage('Pull from github') {
+        git url: 'https://github.com/PGSSoft/php-docker-skeleton'
```

Comment:
better is to use [checkout](https://jenkins.io/doc/pipeline/steps/workflow-scm-step/#checkout-general-scm)